pages/linux/semanage.md

```diff
@@ -0,0 +1,24 @@
+# semanage
+
+> SELinux Policy Management tool.
+> More information: <https://linux.die.net/man/8/semanage>.
+
+- Output local customizations:
+
+`semanage -S {{store}} -o {{output_file}}`
+
+- Input local customizations:
+
+`semanage -S {{store}} -i {{input_file}}`
+
+- Manage booleans. Booleans allow the administrator to modify the confinement of processes based on his configuration:
+
+`semanage boolean -S {{store}} --delete|--modify|--list|--noheading|--deleteall -on|-off -F {{boolean|boolean_file}}`
```

Comment:
Hrm, this looks pretty complicated. Is there any way we can simplify this at all? If not, then I notice some missing token syntax here.
```suggestion
`semanage boolean -S {{store}} {{--delete|--modify|--list|--noheading|--deleteall}} {{-on|-off}} -F {{boolean|boolean_file}}`
```

Remember, the token syntax is for _anything_ the user can change :slightly_smiling_face: